lib/util/MessageUtil.js

```diff
@@ -115,7 +115,27 @@ export function createMessageFlowSemantics(injector, task, participant) {
   let messageFlow = moddle.create('bpmn:MessageFlow', {
     messageRef: message,
     sourceRef: participant,
-    targetRef: task.participantRef.find(p => p != participant)
+    targetRef: task.participantRef.find(p => p !== participant)
+  });
+  messageFlow.id = moddle.ids.nextPrefixed('MessageFlow_', messageFlow);
+
+  return messageFlow;
+}
+
+/**
+ * Creates and returns a message flow without creating a message
+ * @param injector
+ * @param message
+ * @param source_participant
+ * @param target_participant
+ * @returns {bpmn:MessageFlow}
+ */
+export function createMessageFlow(injector, message, source_participant, target_participant) {
```

Comment:
Could this function be used in `createMessageFlowSemantics` as well to reduce code duplication?

Reply:
Yes